servant/src/Servant/API/ContentTypes.hs

```diff
@@ -65,7 +65,6 @@ module Servant.API.ContentTypes
     , AllMime(..)
     , AllMimeRender(..)
     , AllMimeUnrender(..)
-    , eitherDecodeLenient
     , canHandleAcceptH
```

Comment:
Keep `eitherDecodeLenient` as alias of `eitherDecde`, but mark it as deprecated (cf. below).

```suggestion
    , eitherDecodeLenient
    , canHandleAcceptH
```